get_sample_covariates.py

```diff
@@ -85,35 +79,35 @@ def main(
 
     # sex
     # option 1: separate files
-    # tob
-    tob_sample_qc_ht = hl.read_table(tob_sampleqc_file_path)
-    # convert to pandas
-    tob_sample_qc_df = tob_sample_qc_ht.to_pandas()
-    # extract info and reformat to table
-    tob_sample_qc_df['sample_id'] = [str(s) for s in tob_sample_qc_df['s']]
-    # only retain relevant columns
-    tob_sex_df = tob_sample_qc_df[['sample_id', 'sex']]
-    # bioheart
-    bioheart_sample_qc_ht = hl.read_table(bioheart_sampleqc_file_path)
-    # convert to pandas
-    bioheart_sample_qc_df = bioheart_sample_qc_ht.to_pandas()
-    # extract info and reformat to table
-    bioheart_sample_qc_df['sample_id'] = [str(s) for s in bioheart_sample_qc_df['s']]
-    # only retain relevant columns
-    bioheart_sex_df = bioheart_sample_qc_df[['sample_id', 'sex']]
-    # combine_info
-    sex_df = pd.concat([tob_sex_df, bioheart_sex_df], axis=0)
+    # # tob
+    # tob_sample_qc_ht = hl.read_table(tob_sampleqc_file_path)
+    # # convert to pandas
+    # tob_sample_qc_df = tob_sample_qc_ht.to_pandas()
+    # # extract info and reformat to table
+    # tob_sample_qc_df['sample_id'] = [str(s) for s in tob_sample_qc_df['s']]
+    # # only retain relevant columns
+    # tob_sex_df = tob_sample_qc_df[['sample_id', 'sex']]
+    # # bioheart
+    # bioheart_sample_qc_ht = hl.read_table(bioheart_sampleqc_file_path)
+    # # convert to pandas
+    # bioheart_sample_qc_df = bioheart_sample_qc_ht.to_pandas()
+    # # extract info and reformat to table
+    # bioheart_sample_qc_df['sample_id'] = [str(s) for s in bioheart_sample_qc_df['s']]
+    # # only retain relevant columns
+    # bioheart_sex_df = bioheart_sample_qc_df[['sample_id', 'sex']]
+    # # combine_info
+    # sex_df = pd.concat([tob_sex_df, bioheart_sex_df], axis=0)
 
     # # option 2: combined file
     # at the moment this is not up to date, but ideally this would be what we'd run instead
     # sample_qc_ht_path = dataset_path(f'large_cohort/{vds_version}/sample_qc.ht')
-    # sample_qc_ht = hl.read_table(sample_qc_ht_path)
-    # # convert to pandas
-    # sample_qc_df = sample_qc_ht.to_pandas()
-    # # extract info and reformat to table
-    # sample_qc_df['sample_id'] = [str(s) for s in sample_qc_df['s']]
-    # # only retain relevant columns
-    # sex_df = sample_qc_df[['sample_id', 'sex']]
+    sample_qc_ht = hl.read_table(tenk10k_sampleqc_file_path)
+    # convert to pandas
+    sample_qc_df = sample_qc_ht.to_pandas()
+    # extract info and reformat to table
+    sample_qc_df['sample_id'] = [str(s) for s in sample_qc_df['s']]
+    # only retain relevant columns
+    sex_df = sample_qc_df[['sample_id', 'sex']]
```

Comment:
This column field will need to be 'sex_karyotype' to use the genetically the inferred sex.

Reply:
Yup I just noticed and made the change!